workflow/admin.py

```diff
@@ -55,10 +55,10 @@ def queryset(self, request, queryset):
         `self.value()`.
         """
         # Filter by logged in users allowable countries
-        user_countries = getCountry(request.user)
-        return queryset.filter(country__in=user_countries)
-
-    pass
+        user_countries = TolaUser.objects.all().filter(
+            user__id=request.user.id).values('countries')
+        countries = Country.objects.all().filter(id__in=user_countries)
+        return queryset.filter(country__in=countries)
```

Comment:
- You don't need to use `all()` to filter Country and TolaUser.
- You don't need to fetch the countries from the database, you can get a list of country ids with `values_list('countries', flat=True)` and then use it in the queryset filtering.

Reply:
I didn't see any test for it as well

Reply:
True, I just copy pasted the function `getCountry`.

Reply:
Done @jefmoura . I deleted this class and others because they were not used.